docs/gatsby-config.js

```diff
@@ -16,29 +16,43 @@ module.exports = {
           1: 'version-1',
         },
         sidebarCategories: {
-          null: ['index', 'getting-started', 'whats-new'],
-          Essentials: [
-            'essentials/schema',
-            'essentials/server',
-            'essentials/data',
+          null: [
+            'index',
+            'getting-started',
+            '[Changelog](https://github.com/apollographql/apollo-server/blob/master/CHANGELOG.md)',
           ],
-          Features: [
-            'features/caching',
-            'features/mocking',
-            'features/errors',
-            'features/data-sources',
-            'features/subscriptions',
-            'features/metrics',
-            'features/graphql-playground',
-            'features/scalars-enums',
-            'features/unions-interfaces',
-            'features/directives',
-            'features/creating-directives',
-            'features/authentication',
-            'features/testing',
-            'features/apq',
-            'features/health-checks',
-            'features/file-uploads',
+          'Defining a Schema': [
+            'defining-a-schema/schema',
+            'defining-a-schema/scalars-enums',
+            'defining-a-schema/unions-interfaces',
+            'defining-a-schema/directives',
+            'defining-a-schema/creating-directives',
+          ],
+          'Resolving Requests': [
+            'resolving-requests/data',
+            'resolving-requests/data-sources',
+            'resolving-requests/errors',
+            'resolving-requests/authentication',
+            'resolving-requests/file-uploads',
+            'resolving-requests/subscriptions',
+          ],
+          'Testing & Monitoring': [
+            'testing-and-monitoring/mocking',
+            'testing-and-monitoring/graphql-playground',
+            'testing-and-monitoring/testing',
+            'testing-and-monitoring/metrics',
+            'testing-and-monitoring/health-checks',
+          ],
+          'Improving Performance': [
```

Comment:
We don't use Title Case on the individual pages (so the sub-sections of this in the left nav won't use it), but should we also not use it for the top to make them consistent?

Reply:
Ah, good point about the all-caps!  I'm not usually a fan of title-casing, but I know it has some purpose as a visual treatment and I merely drawing attention to the difference.

(As to my not-a-fan-ness: I find title case as a hard visual treatment to contributors to grok/juggle since it's often at the mercy of the number of words and how many of them are connectives/prepositions.)

I'll resolve this, either way!